Package.swift

```diff
@@ -40,7 +40,7 @@ let package = Package(
     .package(
       name: "GoogleUtilities",
       url: "https://github.com/google/GoogleUtilities.git",
-      "7.2.1" ..< "8.0.0"
+      "8.0.0" ..< "9.0.0"
```

Comment:
Needs to point at the branch like https://github.com/firebase/firebase-ios-sdk/blob/auth-swift/Package.swift#L157

    .package(
      url: "https://github.com/google/GoogleUtilities.git",
      branch: "release-8.0"
      // TODO: Update to 8.0.0 when ready.
      // "8.0.0" ..< "9.0.0"
    ),

Reply:
Thanks! Note: I used the deprecated [`.branch("...")`](https://developer.apple.com/documentation/packagedescription/package/dependency/package(url:_:)-4tkwi) syntax since we're using the Swift 5.3 toolchain. I think we should consider increasing that as part of major version bump.